packages/e2e-test-utils-playwright/README.md

```diff
@@ -4,6 +4,10 @@ End-To-End (E2E) Playwright test utils for WordPress.
 
 _It works properly with the minimum version of Gutenberg `9.2.0` or the minimum version of WordPress `5.6.0`._
 
+> ⚠️ **Prerelease Note**
```

Comment:
I just learned (from @Mamaduka, thanks!) that we have an [official recommendation](https://developer.wordpress.org/block-editor/contributors/documentation/#callout-notices) for how to add notes to documentation. Maybe we should try that here?

Reply:
Oh, good to know! Updated to alert style in https://github.com/WordPress/gutenberg/pull/43998/commits/129d6ad8b527c405415f67b755c0bf72c5949198.